bitcoin/src/blockdata/transaction.rs

```diff
@@ -404,6 +405,17 @@ impl Sequence {
     }
 }
 
+impl FromHexStr for Sequence {
+    type Error = crate::parse::ParseIntError;
+
+    fn from_hex_str_no_prefix<S: AsRef<str> + Into<String>>(s: S) -> Result<Self, Self::Error>
+        where Self: Sized
+    {
+        let sequence = crate::parse::hex_u32(s)?;
+        Ok(Self(sequence))
```

Comment:
This is okay for sequence. As a nit and for consistency, still good to use the constructor here.

Reply:
: